Civi/Core/Container.php

```diff
@@ -203,8 +203,12 @@ public function createContainer() {
       'Civi\Token\TokenCompatSubscriber',
       array()
     ))->addTag('kernel.event_subscriber');
+    $container->setDefinition("crm_mailing_action_tokens", new Definition(
+      "CRM_Mailing_ActionTokens",
+      array()
+    ))->addTag('kernel.event_subscriber');
 
-    foreach (array('Activity', 'Contribute', 'Event', 'Member') as $comp) {
+    foreach (array('Activity', 'Contribute', 'Event', 'Mailing', 'Member') as $comp) {
```

Comment:
We do cache the container definitions now-a-days (v4.7+; see `loadContainer()`), but I'd rather not create a sui-generis mechanism just for locating the `CRM_*_Token` classes. Rather, cleaning this bit should wait until we have a generally decent way to locate/discover service definitions (e.g. via annotation or YAML -- as hinted by https://github.com/totten/civicrm-core/blob/e0c753854db4ed1bcd3b0659a29b79f535c1aa08/Civi/Core/Container.php#L112 ).